indico/modules/designer/__init__.py

```diff
@@ -80,13 +80,20 @@ class PageLayout(RichIntEnum):
 
 
 @signals.core.get_placeholders.connect_via('designer-fields')
-def _get_notification_placeholders(sender, **kwargs):
+def _get_designer_placeholders(sender, regform=None, **kwargs):
     from indico.modules.designer import placeholders
     for name in placeholders.__all__:
         obj = getattr(placeholders, name)
         if isinstance(obj, type) and issubclass(obj, placeholders.DesignerPlaceholder) and hasattr(obj, 'name'):
             yield obj
 
+    if regform:
+        from indico.modules.designer.placeholders import RegistrationFormFieldPlaceholder
+        from indico.modules.events.registration.models.items import RegistrationFormItemType
+        for field in regform.active_fields:
+            if field.type != RegistrationFormItemType.field_pd:      # Personal data fields are already included
+                yield RegistrationFormFieldPlaceholder(field=field)  # in the 'registrant' group
```

Comment:
```suggestion
            # Personal data fields are already included in the 'registrant' group
            if field.type != RegistrationFormItemType.field_pd:
                yield RegistrationFormFieldPlaceholder(field=field)
```

not a fan of "block-style" comments ;)